akka-testkit/src/main/scala/akka/testkit/TestActorRef.scala

```diff
@@ -23,7 +23,22 @@ import akka.pattern.ask
  * @since 1.1
  */
 @silent // 'early initializers' are deprecated on 2.13 and will be replaced with trait parameters on 2.14. https://github.com/akka/akka/issues/26753
```

Comment:
Should we move this implementation to the `scala-3` folder and use proper trait parameters in Scala 3?
@raboof @johanandren

Reply:
If that works with minimal changes it sound fine by me